vector/src/main/res/values/strings.xml

```diff
@@ -174,7 +174,8 @@
     <string name="initial_sync_start_importing_account_joined_rooms">Initial sync:\nLoading your conversations\nIf you\'ve joined lots of rooms, this might take a while</string>
     <string name="initial_sync_start_importing_account_invited_rooms">Initial sync:\nImporting invited rooms</string>
     <string name="initial_sync_start_importing_account_left_rooms">Initial sync:\nImporting left rooms</string>
-    <string name="initial_sync_start_importing_account_groups">Initial sync:\nImporting communities</string>
+    <!--TODO:delete-->
```

Comment:
Todo? (a few of this in this file)

Reply:
we do not remove translatable resources, instead we mark them to be removed, to not cause conflicts on Webplate

https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#removing-existing-strings